src/pages/guest/[uid].astro

```diff
@@ -1,95 +1,254 @@
+          <span>
+            {guest.data.name}'s Twitter
+          </span>
+        </a>
+      </div>
+    </div>
+
+    <div class="other-guests-container">
+      <h2>Other Speakers</h2>
+      <div class="other-guests-inner">
+        {
+          nextGuests.map((guest) => (
+            <div class="other-guest">
+              <GuestAvatar guest={guest} />
+              <a class="other-guest-name" href={`/guest/${guest.uid}`}>
+                {guest.data.name}
+              </a>
+            </div>
+          ))
+        }
       </div>
-      <dl>
-        <dt>Bio</dt>
-        <dd>
-          <Fragment set:html="guestBioHTML" />
-        </dd>
-        <dt>Topic</dt>
-        <dd>{guest.data.topicTitle}</dd>
-        <dd>
-          <Fragment set:html="topicDescriptionHTML" />
-        </dd>
-      </dl>
     </div>
   </article>
 </Layout>
 
 <style>
-  #guest-page {
-    --container-width: 60ch;
+  .guest-page {
+    --container-width: 1136px;
 
     margin-inline: auto;
     padding-block-start: var(--space-l-xl);
     width: min(var(--container-width), 100% - var(--space-l));
+    display: grid;
+    grid-template-columns: 1fr;
+    align-items: start;
+    gap: var(--space-l) var(--space-l);
   }
 
   h1 {
-    text-align: center;
     margin-bottom: var(--space-s);
   }
 
   .guest-details {
     display: flex;
-    flex-flow: column;
-    align-items: center;
-    gap: var(--space-2xs);
-    span:last-child {
-      color: var(--header-font-color);
-    }
+    flex-flow: row;
+    gap: 0.5ch;
+    font-size: var(--step-0);
+    text-transform: uppercase;
   }
 
   dl {
-    background: var(--box-color);
-    padding-inline: var(--space-m);
-    padding-block: var(--space-s);
-    border-radius: 0.5rem;
-    margin-top: var(--space-m);
   }
   dt {
     font-size: var(--step-1);
     font-family: var(--header-font);
-    color: var(--header-font-color);
+    color: var(--accent);
     margin-block-start: var(--space-s-m);
+    font-weight: 700;
   }
   dt:first-child {
     margin-block-start: 0;
   }
   dd {
+    margin-block-start: var(--space-2xs);
+    font-size: var(--step-0);
+  }
+
+  .guest-links {
+    display: flex;
+    flex-flow: column;
+    gap: var(--space-s);
+    border-block-start: 1px solid var(--accent);
     margin-block-start: var(--space-s);
+    padding-block-start: var(--space-2xs);
+  }
+
+  .guest-link {
+    display: flex;
+    color: var(--body-font-color);
+    gap: var(--space-2xs);
+    text-decoration: none;
+    font-weight: 500;
+    font-size: var(--step-0);
+  }
+
+  .link-icon {
+    width: var(--step-0);
+    color: var(--accent);
+  }
+
+  .other-guests-container {
+    margin-block-start: var(--space-3xl);
+    gap: var(--space-l);
+    display: grid;
+  }
+
+  .other-guests-inner {
+    display: grid;
+    width: 100%;
+    grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+    gap: var(--space-xl) var(--space-l);
+  }
+
+  .other-guest {
```

Comment:
I love the idea of featuring other speakers! I do notice that the based on where the other speakers section is on the page, the white text against this particular part of the gradient is about 2.7:1 at best, and around 2:1 at most difficult (hard to give an exact number, because gradients)

<img width="396" alt="Screenshot of Cassidy Williams's avatar and name in the Other Speakers section against a peach-colored section of the background gradient." src="https://user-images.githubusercontent.com/18060369/205809614-4f8d24e2-6975-47d2-b4f4-9ccadc179080.png">
<img width="396" alt="Maggie Appleton and Wes Bos's avatars and names in a column at mobile width. Wes's name is against a very light peachy color." src="https://user-images.githubusercontent.com/18060369/205810405-ef9e6930-2f15-4558-9452-db29b31edc54.png">

Given the speaker names can appear at font sizes too small to meet WCAG's large text definition, we need to target a contrast ratio of 4.5:1. Maybe we put the translucent purple backgrounds on them like we do the contributor cards?

Reply:
Thanks for spotting that! I should have mentioned that in my pull request, instead I put it in a random tweet 😄

Just pushed a change that keeps this gradient on the homepage but changes it on other pages like this one. Should handle the issue entirely
![image](https://user-images.githubusercontent.com/19617280/205911491-6a2db335-ea17-4d98-897f-678d128441ab.png)

Reply:
Nice! The _lowest_ content ratio I was getting was about 8.8:1, which well exceeds the required 4.5:1. This works really nicely!

Reply:
Awesome, thanks for measuring that! Resolving this convo.